README.md

```diff
@@ -62,13 +62,40 @@ Your callback will be called once with the arguments
     or [`XDomainRequest`][4] instance (if on IE8/IE9 &&
     `options.useXDR` is set to `true`)
  - `headers`: A collection of headers where keys are header names converted to lowercase
-    
 
-Your callback will be called with an [`Error`][5] if there is an error in the browser that prevents sending the request. 
-A HTTP 500 response is not going to cause an error to be returned. 
-    
-If `options` is a string then it's a short hand for
-    `{ method: "GET", uri: string }`
+
+Your callback will be called with an [`Error`][5] if there is an error in the browser that prevents sending the request.
+A HTTP 500 response is not going to cause an error to be returned.
+
+## Shorthands
+
+## `var req = xhr(url, callback)`
+`xhr` may also be called with a simple string instead of a set of options. In this case, a GET request will be made to that url.
+
+## `var req = xhr(url, options, callback)`
+The above shorthand may also be called with the standard set of options.
```

Comment:
I just mean this signature is not a shorthand so I think it shouldn't be listed in this section